src/main/kotlin/uk/gov/justice/digital/hmpps/sendlegalmailtoprisonsapi/config/WebClientConfiguration.kt

```diff
@@ -32,9 +41,4 @@ class WebClientConfiguration(
       next.exchange(filtered)
     }
   }
-
-  @Bean
```

Comment:
This bean wasnt being used

Reply:
Oh... it probably should be used on the /health page

Reply:
I'll add it back for the time being in that case, and we can look at the health endpoint and whether it is/should be using this bean another day 👍